www/class/centreon-partition/partEngine.class.php

```diff
@@ -316,8 +299,18 @@ private function getLastPartRange($table, $db)
         }
         $row = $dbResult->fetch();
 
+        $lastPart = 0;
+        // dont care of MAXVALUE
```

Comment:
MAXVALUE can be hidden directly in the query:
```
SELECT MAX(CONVERT(PARTITION_DESCRIPTION, SIGNED INTEGER)) as lastPart
FROM INFORMATION_SCHEMA.PARTITIONS
WHERE TABLE_NAME=?
AND TABLE_SCHEMA=?
AND PARTITION_DESCRIPTION != 'MAXVALUE'
GROUP by TABLE_SCHEMA,TABLE_NAME;
```

Reply:
we don't use that query anymore. We won't use that request because it costs a lot.